cli/shortcuts/shortcuts.go

```diff
@@ -0,0 +1,21 @@
+package shortcuts
+
+import "github.com/buildbuddy-io/buildbuddy/cli/arg"
+
+var (
+	shortcuts = map[string]string{
+		"b": "build",
+		"t": "test",
+		"q": "query",
+		"r": "run",
+		"f": "fix",
+	}
+)
+
+func HandleShortcuts(args []string) []string {
+	command, idx := arg.GetCommandAndIndex(args)
```

Comment:
Use `parser.GetBazelCommandAndIndex` here, otherwise I think this might fail for things like `bazel --output_base foo b blah`

Reply:
That would work for most of these, but not for `fix`. And it wouldn't work for `bazel --output_base test b blah`.

I added a change to canonicalize just startup options very early on in `bb.go`. 

In order to do this, I hardcoded just the set of startup options that take non-boolean options, which is not very large and I don't suspect will change very often. In the unlikely case these change, we can just add the new flags. Also for a new flag to cause an issue, the user would have to:
a) use this new flag 
b) use buildbuddy cli 
c) use a shortcut 
d) use the non `=` version of the flag

In order for there to be an issue, which further reduces the likelihood of this being a problem at which point we would just add the new flag.

This should solve this problem for all places we call `GetBazelCommand` and it's variants.

Let me know what you think.